File: Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Indexes/StartIndexing.cs
using Raven.Client.Documents;
using Raven.Client.Documents.Operations.Indexes;
using Raven.Client.ServerWide.Operations;

namespace Raven.Documentation.Samples.ClientApi.Operations.Indexes
{
	public class StartIndexing
	{
		private interface IFoo
		{
            /*
            #region start_1
            public StartIndexingOperation()
            #endregion
            */
        }

        public StartIndexing()
		{
			using (var store = new DocumentStore())
			{
                #region start_2
                // Start indexing
                store.Maintenance.Send(new StartIndexingOperation());

                // Disable the database
                store.Maintenance.Server.Send(new ToggleDatabasesStateOperation(store.Database, true));
                // Enable the database to apply new settings
                store.Maintenance.Server.Send(new ToggleDatabasesStateOperation(store.Database, false));
                #endregion
            }
		}
	}
}

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Maintenance/Indexes/StartIndexing.java
package net.ravendb.ClientApi.Operations.Indexes;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.operations.indexes.StartIndexingOperation;

public class StartIndexing {

    private interface IFoo {
        /*
        //region start_1
        public StartIndexingOperation()
        //endregion
        */
    }

    public StartIndexing() {
        try (IDocumentStore store = new DocumentStore()) {
            //region start_2
            store.maintenance().send(new StartIndexingOperation());
            //endregion
        }
    }
}
